api/src/main/java/ca/bc/gov/educ/api/trax/service/institute/CodeService.java

```diff
@@ -37,6 +43,8 @@ public class CodeService {
 	SchoolFundingGroupCodeTransformer schoolFundingGroupCodeTransformer;
 	@Autowired
 	private RestUtils restUtils;
+	@Autowired
+	RedisTemplate<String, String> redisTemplate;
 
 	public List<SchoolCategoryCode> getSchoolCategoryCodesFromInstituteApi() {
 		try {
```

Comment:
Create a webclient and qualifier for grad-edx-client like the pattern established [here](https://github.com/bcgov/EDUC-GRAD-TRAX-API/blob/main/api/src/main/java/ca/bc/gov/educ/api/trax/config/GradTraxConfig.java#L56) and used [here](https://github.com/bcgov/EDUC-GRAD-TRAX-API/blob/main/api/src/main/java/ca/bc/gov/educ/api/trax/util/CommonSchoolCache.java#L29) and configured [here](https://github.com/bcgov/EDUC-GRAD-TRAX-API/blob/main/api/src/main/resources/application.yaml#L47).

Reply:
Fixed https://github.com/bcgov/EDUC-GRAD-TRAX-API/blob/c79bdcc841ed4529314fd9ef0d3bd821fb053b15/api/src/main/java/ca/bc/gov/educ/api/trax/service/institute/CodeService.java#L31